packages/ingest/readers/github.py

```
@@ -169,3 +203,518 @@ def _get_default_branch(self, owner: str, repo: str) -> str | None:
+                    )
+                )
+
+        return entities
+
+    def _extract_documentation(
+        self,
+        client: httpx.Client,
+        headers: dict[str, str],
+        owner: str,
+        repo: str,
+        branch: str,
```

Ruff (0.14.2)

Unused method argument: `branch`

(ARG002)